Python-packages/covidcast-py/covidcast/plotting.py

```diff
@@ -79,7 +80,8 @@ def plot(data: pd.DataFrame,
       Defaults to `True`.
     :param kwargs: Optional keyword arguments passed to ``GeoDataFrame.plot()``.
     :param plot_type: Type of plot to create. Either choropleth (default) or bubble map.
-    :return: Matplotlib figure object.
+    :param ax: Optional matplotlib axis to plot on.
```

Comment:
I think the docstring should say what happens if `ax` is not provided (axes are made, states are plotted). Maybe that goes in the text above, such as in a paragraph after the one-line summary explaining that the maps are plotted on US states by default.

Reply:
I think I had a bug here, the background states are always plotted and I had left that statement out. I've also added a sentence describing the return behavior.